Jenkinsfile

```diff
@@ -667,6 +669,11 @@ pipeline {
                     post {
                         success {
                             buildRpmPost condition: 'success'
+                            script {
+                                Map stage_info = parseStageInfo()
+                                    stash name: stage_info['target'] + '-required-mercury-rpm-version',
+                                          includes: stage_info['target'] + '-required-mercury-rpm-version'
+                            }
```

Comment:
I mentioned when I gave you this code that this was just PoC that should have gone into `pipeline-lib:buildRpmPost()` for production.  This sort of thing is going to blow up our `Jenkinsfile` again.